test/service/test_tdr.py

```diff
@@ -323,6 +323,7 @@ def convert_metadata(self, metadata: MutableJSON) -> MutableJSON:
         """
 
         metadata = copy.deepcopy(metadata)
+        metadata['links.json']['schema_type'] = 'links'  # V1 uses 'link_bundle'
```

Comment:
I think this comment will confuse us in the future. What is V1 and why do we need to point out a thing of the past?

Reply:
"DCP v1." 

To save us from writing out the entire contents of the expected output from the TDR plugin's bundle emulation, the tests load an old bundle and mutate it to reflect the changes in DCP v2. There are many several similar comments throughout the `_convert_metadata` and `_convert_manifest` methods, explaining what inconsistencies are being addressed.

Reply:
Makes sense. "DCP/1" would be more descriptive.

https://github.com/DataBiosphere/azul/issues/2374